components/automate-ui/src/app/pages/user-management/user-management.component.ts

```diff
@@ -129,4 +137,11 @@ export class UserManagementComponent implements OnInit, OnDestroy {
         }
     });
   }
+
+  public showEmptyStateMessage(): boolean {
+    this.loading$.subscribe((loadingStatus: boolean) => {
+      return !loadingStatus || this.users.length === 0;
+    });
+    return false;
```

Comment:
❓ When will this line be executed?

Reply:
Yeah this whole function looks strange to me. Seems like we'd want to set the result of the subscribe to a field that is defaulted to false, put the subscribe in the initialization code, and simply return the field here?

Reply:
In fact, I would exactly mirror the team-details version, which follows along Tyler's suggestion:
```
  showEmptyStateMessage(): boolean {
    return !this.isLoading && this.users.length === 0;
  }
```
Then get rid of `loading$` completely; there is a lot duplication in this file between that and `userStatus$`--no need for both of them.
Finally, change the block on L132-138 to this:
```
    this.userStatus$.pipe(
      takeUntil(this.isDestroyed))
      .subscribe((status) => {
        this.isLoading = status === EntityStatus.loading;
        if (!this.isLoading) {
          this.closeCreateModal();
        }
    });
```

Reply:
Yeah we will revisit this. We were getting an odd compilation error without it.